src/directives/layertree.js

```diff
@@ -75,6 +75,11 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
  * controller: "layertreeCtrl". You can refer to that property in a custom
  * template for example.
  *
+ * @htmlAttribute {???} ngeo-layertree The tree
+ * @htmlAttribute {string} ngeo-layertree-templateurl The template url
+ * @htmlAttribute {ol.Map} ngeo-layertree-map The map
+ * @htmlAttribute {???} ngeo-layertree-nodelayer ???
```

Comment:
{string} (expression that will be parsed, like "gmfLayertreeCtrl.getLayer(node, depth)")